src/NServiceBus.AcceptanceTests/EndpointTemplates/TestSuiteConstraints.cs

```diff
@@ -9,6 +9,8 @@ public interface ITestSuiteConstraints
         bool SupportNativePubSub { get; }
 
         bool SupportNativeDeferral { get; }
+
+        bool ProvideOutboxPersistence { get; }
```

Comment:
should we then align the other constraints to be `Supports` too, or should it be `SupportOutbix`?